nshlib/nsh_netcmds.c

```diff
@@ -959,6 +972,9 @@ int cmd_ifconfig(FAR struct nsh_vtbl_s *vtbl, int argc, char **argv)
 #ifdef CONFIG_NET_IPv4
   UNUSED(gip);
 #endif
+#ifdef CONFIG_NET_IPv6
+  (void)gip6;
```

Comment:
```suggestion
  UNUSED(gip6);
```
Also please add same above.